rules/_deprecated/privilege_escalation_setgid_bit_set_via_chmod.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/04/23"
+maturity = "deprecated"
+updated_date = "2021/03/16"
```

Comment:
Should we include the deprecation date in the TOML too?
I think updated_date should match. No need to change a rule after it's deprecated.

Reply:
good idea